packages/@o3r/schematics/src/tasks/ng-add/index.ts

```diff
@@ -26,6 +26,9 @@ export interface NgAddPackageOptions {
    * Type of dependency to install
    */
   dependencyType?: NodeDependencyType;
+
+  /** Flag to skip the execution of ng add and only install the package. Used mostly for external packages */
+  skipNgAddSchematicRun?: boolean;
```

Comment:
Is it already part of the `schema.json` (which is not part of this PR)?

Reply:
For the moment is just a parameter of the **ngAddPackages** function. 
Passing it as an option will be done in [691](https://github.com/AmadeusITGroup/otter/issues/691)